src/datapump/jobs/geotrellis.py

```diff
@@ -280,7 +349,8 @@ def _get_result_tables(self) -> List[AnalysisResultTable]:
         LOGGER.debug(f"Looking for analysis results at {result_path}")
         resp = get_s3_client().list_objects_v2(Bucket=bucket, Prefix=prefix)
 
-        LOGGER.debug(resp)
+        # FIXME: Do we really need to log this?
```

Comment:
Sorry, I think I added this to debug an issue a week or two ago, not sure how this go in there.